### PR TITLE
Added search feature to notes editor; interworks with global search

### PR DESCRIPTION
**Pull request by stebeh**
I added a feature that allows to search text within a note. Previously, the search function would only show which notes contain a certain text, but not where exactly within a note the search term was found. The new version allows to jump from one occurrence in the text to the next. I found this essential because I have notes that are hundreds of lines long, and searching "by eye" was extremely cumbersome.

### CODE REVIEW

File: NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java [hunk 1]
```diff
@@ -147,7 +147,7 @@ protected void onListItemClick(ListView l, View v, int position, long id) {
 			Uri uri = ContentUris.withAppendedId(Notes.CONTENT_URI, id);
 
 			// Launch activity to view/edit the currently selected item
-			startActivity(new Intent(Intent.ACTION_EDIT, uri));
+			startActivity(new Intent(Intent.ACTION_EDIT, uri).putExtra("SEARCH_STRING", queryString));
```
Comment (friedger) [inline NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java]: This should go into a separate line

File: NotePad/res/layout/note_editor.xml [hunk 2]
```diff
@@ -16,6 +16,14 @@
  * limitations under the License.
  -->
 
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (friedger) [inline NotePad/res/layout/note_editor.xml]: It would be nice to have a screenshot in the pull request to see how it looks like

Comment (stebeh) [inline NotePad/res/layout/note_editor.xml]: screenshot from 2015-01-18 18 22 43

File: NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java (hunk 1 above)
Comment (stebeh) [inline NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java]: Fixed in another commit to the search branch on stebeh/notepad.

File: NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java [hunk 3]
```diff
@@ -147,7 +147,8 @@ protected void onListItemClick(ListView l, View v, int position, long id) {
 			Uri uri = ContentUris.withAppendedId(Notes.CONTENT_URI, id);
 
 			// Launch activity to view/edit the currently selected item
-			startActivity(new Intent(Intent.ACTION_EDIT, uri));
+			startActivity(new Intent(Intent.ACTION_EDIT, uri)
```
Comment (friedger) [inline NotePad/src/org/openintents/notepad/search/SearchQueryResultsActivity.java]: I wanted to have a variable for that. But LGTM.

**Pull request merged**